.github/workflows/ci.yaml

```diff
@@ -52,6 +52,26 @@ jobs:
             ui/**/dist
             !ui/node_modules
 
+  build-storybook:
```

Comment:
Currently, this builds storybook and outputs it as an artifact. It'll be nice to publish it to github pages in the future, but I want to tackle that in a follow-up, ideally after we've fleshed out more stories.